### ISSUE TITLE
Keyless driving toggle 

### ISSUE THREAD

**Issue by doctorfagan** [Kemmey/Kemmey-TeslaWatch-Public#269]
Hi, I can’t get my keyless driving toggle to appear in my Apple Watch series 6 cellular even though I have keyless driving enabled in my 2020 Tesla model 3 performance ? My iPhone unlocks the car fine when close

**Comment by Kemmey** [comment]
Hey @doctorfagan,

Have you activated Remote start and entered your Tesla account password in the settings menu?

Does Remote start work in the official Tesla iOS app?

**Comment by doctorfagan** [comment]
Yes it works in the official Tesla app. I have entered login details on watch but they never seem to disappear as if there should be an enter button but I can’t see one 

Rgds

Darren Timoney

**Comment by Kemmey** [comment]
They should render as dots. Once you see them as dots, you're good to go. 

Then the Remote start button should show up under controls.

**Comment by doctorfagan** [comment]
They dots are showing but no keyless driving toggle -

**Comment by Kemmey** [comment]
@doctorfagan So you see something similar as the below in settings:

![image](https://user-images.githubusercontent.com/4688084/105208755-57e07480-5b49-11eb-828e-dafbde68a9de.png)

And then from main screen, swipe right to see all controls, then you should see below button:

![image](https://user-images.githubusercontent.com/4688084/105208821-6c247180-5b49-11eb-8de3-e655695b29e3.jpeg)

**Comment by doctorfagan** [comment]
Yes I have done that now but i still cannot see the toggle for keyless driving ?

**Comment by Kemmey** [comment]
@doctorfagan Do you see the Remote start button circled in blue?

**Comment by doctorfagan** [comment]
Yes and I have activated it but still no toggle for keyless driving below password entry in settings

**Comment by Kemmey** [comment]
I'm not sure what you mean?! Then remote start is working just fine?! Have you checked in your car that you are able to drive? Try leaving your phone and go to your car, tap unlock, then remote start. Your car should show that it is ready to drive.

**Comment by doctorfagan** [comment]
Does the keyless driving toggle that appears in other peoples Apple Watch app not allow you to walk up to your Tesla without your phone and the car unlock when it senses the watch just like it does with the iPhone ?

**Comment by Kemmey** [comment]
No. That is not possible. Only Tesla can do that.

**Comment by doctorfagan** [comment]
Ok thanks

**Comment by doctorfagan** [comment]
https://youtu.be/8l-I10Yubmk

???

**Comment by Kemmey** [comment]
Yes? That's my app? What do you mean to say? He does not open the car just by having his watch present. He taps the unlock button.

**Comment by doctorfagan** [comment]
Thanks kemmy, I never seen him press the button but must be my mistake. Can you tell me why I cannot see the keyless driving toggle in my watch app ?

**Comment by Kemmey** [comment]
Remote start == Keyless drive
